src/posix/platform/system.cpp

```diff
@@ -127,26 +129,11 @@ void otSysSetInfraNetif(const char *aInfraNetifName, int aIcmp6Socket)
 }
 #endif
 
-void platformInit(otPlatformConfig *aPlatformConfig)
+void _platformInitCoreMode(otPlatformConfig *aPlatformConfig)
```

Comment:
Why does it start with an underscore?

Reply:
This means the method is only an internal method. I use it here to indicate that the high level will only be aware of `platformInit` but will know `_platformInitCoreMode` and `_platformInitOffloadMode`. I think it doesn't matter here. Adding or removing the underscore are both ok.